### PR TITLE
Fetch PRs for each repo serially to avoid rate limit issues

### PR DESCRIPTION
**Pull request by maxbrunsfeld**
/cc @daviwil @jasonrudolph

### CODE REVIEW

File: src/changelog.js [hunk 1]
```diff
@@ -414,8 +414,7 @@ async function getFormattedPullRequestsForDependencies({owner, repo, fromTag, to
     }
   }
 
-  const results = await Promise.all(resultPromises)
-  return results.join('\n\n')
+  return results.join('\n')
```
Comment (daviwil) [inline src/changelog.js]: Does this result in poor spacing between the package sections or does it seem OK?

**Review by daviwil — APPROVED** [review]
bewm :boom:

File: src/changelog.js (hunk 1 above)
Comment (maxbrunsfeld) [reply src/changelog.js]: Wow that's weird, I didn't mean to do that! Good catch.

**Comment by zeke** [comment]
bewm

**Pull request merged**